docs/testing-components/unit-testing/index.md

```diff
@@ -115,41 +115,27 @@ and outputs we can test basically any JavaScript function that returns a value.
 
 ## Testing React
 
-To test react we use [Enzyme](https://enzymejs.github.io/enzyme/) plus other tools you can find out about [here](../test-driven-development/index.md).
+To test react we use [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) plus other tools you can find out about [here](../test-driven-development/index.md).
 
 ```js
 const Text = (props) => {
 	return <p>{props.content}</p>;
 }
 
 test('Should contain text', () => { 
-	const subject = shallow( 
+	render( 
 		<Text content='sample' /> 
 	);
 	
-	const expected = 'sample'; 
-	const actual = subject.text()
-	expect(actual).toBe(expected); 
+	const textElement = screen.querryByText('sample');
+	
+	expect(textElement).not.toBeNull();
 });
 ```
-
-If you wish to learn more about Enzyme's library of functions look [here](https://github.com/enzymejs/enzyme).
-
-The three main parts about Enzyme that you need to know are it's rendering methods.
-
-### shallow()
-
-[Shallow](https://github.com/enzymejs/enzyme/blob/master/docs/api/shallow.md) is the virtual DOM representation. It will only render the component plus one level of children. This allows
-us to stay within the smaller confines of a component when testing.
-
-### mount()
-
-[mount](https://github.com/enzymejs/enzyme/blob/master/docs/api/mount.md) is the virtual DOM representation. It will render everything inside the component, including all nested children.
-This is a little beyond unit testing as you start to test the integration of a few components.
+React Testing Library has access to a single rendering method.
```

Comment:
How about introduce `render`, `rerender`, and `unmount` for users?
```suggestion
React Testing Library provides the following methods regarding render.
```

Reply:
Done